source/NeutronaWand/Preferences.h

```diff
@@ -114,24 +115,24 @@ void readEEPROM() {
     objConfigEEPROM obj_config_eeprom;
     EEPROM.get(i_eepromAddress, obj_config_eeprom);
 
+    // Assume that the VG_MODE as default, overriding as necessary based on stored flags.
+    FIRING_MODE = VG_MODE;
+
     if(obj_config_eeprom.cross_the_streams > 0 && obj_config_eeprom.cross_the_streams != 255) {
```

Comment:
I agree, but I'm hesitant to change the values that are being stored on user's devices. Since the struct is set with separate values I didn't want to muck with this at the moment.

Reply:
Sounds good. We can reconsider the values here when the big menu refactor happens. That might be an excellent excuse to change to structs generally for stuff like this too (for example, so that items changed in the wand menu don't affect the EEPROM menu).

Reply:
That's a good point. We may find there's a better way to stash these values in the future when we refactor. I like that the struct approach gives you more of an OO pattern, where you know the value belongs to an object property which helps namespace everything.